app/models/friendship.rb

```diff
@@ -0,0 +1,8 @@
+class Friendship < ApplicationRecord
+  validates :status, presence: true
```

Comment:
It makes sense to have these two tables separate from one another because we want a separate entity to access friendships and another to access friend_requests. It makes manipulation much easier as we wish to extend other features to this application!

Reply:
@fdezcaminero I understand this, I personally prefer this over having one table for handling friendships and requests. I mean what is purpose of having `status` column in the friendships table? since it will be always `true`.

Reply:
That's right. But in the future, we want to implement a feature where we can show past friendships. In this feature, when a user removes a friend the status changes to false. Access of friends will only be through friendships where the status of that friendship is true.